lib/commands/flutter.dart

```diff
@@ -26,16 +27,22 @@ class FlutterCommand extends Command {
   }
 
   Future<void> run() async {
-    if (!await kLocalFlutterLink.exists()) {
+    final flutterProjectLink = projectFlutterLink();
+
+    if (flutterProjectLink == null) {
+      throw Exception('No FVM config found. Create with <use> command');
+    }
+
+    if (!await flutterProjectLink.exists()) {
       throw Exception('No FVM config found. Create with <use> command');
     }
 
-    final targetLink = File(await kLocalFlutterLink.target());
+    final targetLink = File(await flutterProjectLink.target());
 
-    if (!await targetLink.exists()) {
+    if (!await flutterProjectLink.exists()) {
       throw Exception('Issue found with FVM config. Create with <use> command');
     }
-    await processRunner('./fvm', argResults.arguments,
+    await processRunner(targetLink.path, argResults.arguments,
```

Comment:
Should this be `flutterProjectLink.path`?

Reply:
`flutterProjectLink` is fvm path.
`flutterProjectLink.target()` is real file, it's the flutter file, such as `~/sdk/flutter/beta/bin/flutter`.

Both should be the same.

Reply:
Makes sense!

Reply:
I can add a test to this section to prevent future regression bugs.